### PR TITLE
Update BetterModelClassDiagram and DeleteSequence Diagram

### PR DESCRIPTION
**Pull request by xuelinglow**

### CODE REVIEW

**Review by cliftonchee — APPROVED** [review]
LGTM! Thanks @xuelinglow!

File: docs/diagrams/BetterModelClassDiagram.puml [hunk 1]
```diff
@@ -20,4 +24,13 @@ Person *--> DateOfBirth
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+
+Appointment *--> Nric
+Appointment *--> Date
+Appointment *--> TimePeriod
```
Comment (dayn-1) [inline docs/diagrams/BetterModelClassDiagram.puml:30]: Maybe can add that `TimePeriod` contains 2 `Time`

**Review by dayn-1 — APPROVED** [review]
LGTM!

**Review by cliftonchee — APPROVED** [review]
LGTM! Thanks @xuelinglow

**Pull request merged**